node_modules/oae-core/manageaccess/js/manageaccess.js

```diff
@@ -175,7 +179,12 @@ define(['jquery', 'oae.core', 'jquery.autosuggest'], function($, oae) {
             infinityScroll.removeItems(principalId);
 
             // Mark the member as deleted
-            membersUpdates[principalId] = false;
+            if (pendingMembers[principalId]) {
+                delete membersUpdates[principalId];
+                pendingMembers[principalId] = false;
+            } else {
+                membersUpdates[principalId] = false;
+            }
```

Comment:
Right. I can't think of a clean way to combine the objects (without changing the API utility code).